Detectors/CTP/workflow/src/RawDecoderSpec.cxx

```diff
@@ -119,49 +121,134 @@ void RawDecoderSpec::run(framework::ProcessingContext& ctx)
       size_gbt = 0;
       orbit0 = triggerOrbit;
     }
-    // LOG(info) << "payload size:" << payload.size();
+    // Create 128 bit words
+    gsl::span<const uint8_t> payload(it.data(), it.size());
+    gbtword128_t gbtWord128 = 0;
+    int wordCount = 0;
+    std::vector<gbtword128_t> gbtwords128;
+    LOG(info) << "payload size:" << payload.size();
```

Comment:
I guess all these logs were for the debugging, please reduce the verbosity in the final version

Reply:
yes